src/components/TimePicker/TimePicker.tsx

```diff
@@ -403,8 +393,8 @@ function TimePicker({forwardedRef, defaultValue, onSubmit, onInputChange}) {
     useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.ARROW_RIGHT, arrowRightCallback, arrowConfig);
 
     const handleFocusOnBackspace = useCallback(
-        (e) => {
-            if (selectionMinute.start !== 0 || selectionMinute.end !== 0 || e.key !== 'Backspace') {
+        (e: NativeSyntheticEvent<KeyboardEvent>) => {
+            if (selectionMinute.start !== 0 || selectionMinute.end !== 0 || e.nativeEvent.key !== 'Backspace') {
```

Comment:
Was there a bug related to Backspace?

Reply:
type bug yep. typescript-wise `e.key` doesn't exist. I think this way is the reliable way and We already use it here 
https://github.com/Expensify/App/blob/c51602fd03e66fa24e3b024dcf855840ece5b0ab/src/components/TimePicker/TimePicker.js#L458-L459